mdstat.go

```diff
@@ -43,6 +43,7 @@ type MDStat struct {
 }
 
 // ParseMDStat parses an mdstat-file and returns a struct with the relevant infos.
+// TODO: this should just be called "MDStat" and use a separate parseMDStat function
```

Comment:
Nit: this comment will appear in godoc documentation, consider moving it to line 48 as the first line of the function body instead.

Reply:
I implemented the changes and removed the TODO.